README.rst

```diff
@@ -19,21 +19,53 @@
    
 |doc-img| |covenant-img| |gitter-img| |doi|
 
-**Coqui STT** (🐸STT) is an open-source deep-learning toolkit for training and deploying speech-to-text models. 🐸STT is battle tested in both production and research 🚀
+`👉 Subscribe to 🐸Coqui.ai Newsletter <https://coqui.ai/?subscription=true>`_
 
-**Documentation** for installation, usage, and training models are available on `stt.readthedocs.io <https://stt.readthedocs.io/>`_.
+**Coqui STT** (🐸STT) is a fast, open-source, multi-platform, deep-learning toolkit for training and deploying speech-to-text models. 🐸STT is battle tested in both production and research 🚀
 
-For the **latest release**, including pre-trained models and checkpoints, `see the latest release on GitHub <https://github.com/coqui-ai/STT/releases/latest>`_.
+🐸STT features
+---------------
 
-For contribution guidelines, see `CONTRIBUTING.rst <CONTRIBUTING.rst>`_.
+* High-quality pre-trained STT model.
```

Comment:
s/model/models/

Reply:
but we just have a model now

Reply:
We have Mandarin but not a very good idea of how high-quality it is. Few test sets and not enough Mandarin knowledge